### PR TITLE
Bar-chart-update and Fixed Unknown Aesthetics Warning

### PR DESCRIPTION
**Pull request by ReddyYesh**
Summary of Changes:
Improved Bar Chart Visualization:

Resolved the issue where overlapping bars (original vs. new costs) made it difficult to interpret the data.
Adjusted bar widths and added labels to improve readability.
Fixed Unknown Aesthetics Warning:

The warning regarding unknown aesthetics: text was resolved by properly mapping the text aesthetic to the tooltip in the ggplotly conversion.
This ensures no warnings are displayed when running the visualization code.
Reason for Changes:
Bar Chart Improvement: The previous overlapping design obscured key data, making it hard to compare original and new costs effectively.
Warning Resolution: Eliminating the aesthetics warning improves code quality and ensures smooth execution without unnecessary messages.
Visual Changes:
The updated chart now displays bars side by side, with clear labels and tooltips for easier interpretation.

### CODE REVIEW

**Comment by sanjmeh** [comment]
Will check today. Thanks.

**Review by sanjmeh — CHANGES_REQUESTED** [review]
unfortunately the code has diverged. So you need to do a `git rebase origin/main`. 
in case it doesn't work, please reform from the main and send a new PR.

File: server.R [hunk 1]
```diff
@@ -1,1134 +1,2286 @@
...
+        autosize = TRUE,
+        showlegend = TRUE
+      )
+    
+    fig
+  })
+  
+}
+
+# Ensure the latest version of gt is installed
```
Comment (bkowshik) [inline server.R:1152]: @ReddyYesh This is the start of the duplicate copy.

**Review by bkowshik — CHANGES_REQUESTED** [review]
Thank you for this PR @ReddyYesh 

I see that there is a duplicate copy in `server.R`, the same code is copied twice, could be by mistake. Can you please clean up this PR, removing the duplicate code. Please post here once it's done and this Pull Request is ready for our review.

**Pull request closed without merging**